controllers/apply/form-router-next/steps.js

```diff
@@ -207,7 +207,7 @@ module.exports = function(formId, formBuilder) {
                  * - Pass the full data object from validationResult to the view. Including invalid values.
                  * Otherwise, find the next suitable step and redirect there.
                  */
-                if (errorsForStep.length > 0) {
+                if (errorsForStep.length > 0 && !(req.body.previousBtn || req.body.nextBtn)) {
```

Comment:
I reckon we need to name this conditional, mind making a function something like `shouldRenderErrors` or `shouldProceedWithoutValidation`?

Reply:
Good shout. I will name this function appropriately.